stripe-patrons-data/src/main/scala/com/gu/patrons/services/ParameterStoreService.scala

```diff
@@ -11,9 +11,8 @@ import com.amazonaws.services.simplesystemsmanagement.{
   AWSSimpleSystemsManagementAsync,
   AWSSimpleSystemsManagementAsyncClientBuilder,
 }
-import com.gu.aws.{AwsAsync, CredentialsProvider}
+import com.gu.aws.{AwsAsync, CredentialsProviderDEPRECATEDV1}
```

Comment:
could have deprecated the old one, but I decided to give it this snappy name instead.